libraries/standard/mqtt/test/access/iot_test_access_mqtt_api.c

```diff
@@ -33,6 +33,13 @@ _mqttConnection_t * IotTestMqtt_createMqttConnection( bool awsIotMqttMode,
                                                       const IotMqttNetworkInfo_t * pNetworkInfo,
                                                       uint16_t keepAliveSeconds );
 
+void IotTestMqtt_mqttOperation_tryDestroy( void * pData );
+
+IotNetworkError_t IotTestMqtt_createNetworkConnection( const IotMqttNetworkInfo_t * pNetworkInfo,
+                                                       IotNetworkConnection_t * pNetworkConnection,
+                                                       bool * pCreatedNewNetworkConnection );
+
+
```

Comment:
why are these functions forward declared? Include of iot_test_access_mqtt.h will do that.

Reply:
I agree. It was already there and I just kept the same format.